Gemfile.jruby-2.5.lock.release

```diff
@@ -138,7 +137,7 @@ GEM
     jruby-jms (1.3.0-java)
       gene_pool
       semantic_logger
-    jruby-openssl (0.10.4-java)
+    jruby-openssl (0.10.5-java)
```

Comment:
we'll have to keep jruby-openssl at 0.10.4 here. related to https://github.com/elastic/logstash/pull/12300

```suggestion
    jruby-openssl (0.10.4-java)
```

Reply:
Good point @jsvd